app/assets/stylesheets/personal_info.scss

```diff
@@ -1,3 +1,3 @@
-// Place all the styles related to the CvUserMainDetails controller here.
+// Place all the styles related to the PersonalInfo controller here.
```

Comment:
@AkshayPashnathSuresh 
we don't need this file as we are not planning to add any scss data right now. 
when you need, create one.

Reply:
@AkshayPashnathSuresh 
make sure that all requested changes are done before reassigning it to me 🗡️ 

> we don't need this file as we are not planning to add any scss data right now.
 means delete the file.